omod-2.0/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs2_0/ConceptAttributeTypeController2_0Test.java

```diff
@@ -121,4 +121,21 @@ public void shouldGetAConceptAttributeTypeByUuid() throws Exception {
     	Assert.assertEquals(conceptAttributeType.getUuid(), PropertyUtils.getProperty(result, "uuid"));
     	Assert.assertEquals(conceptAttributeType.getName(), PropertyUtils.getProperty(result, "name"));
     }
+
+    @Test
+    public void shouldRetireAConceptAttributeType() throws Exception {
+        final String UUID = "9516cc50-6f9f-11e0-8414-001e378eb67e";
+        ConceptAttributeType conceptAttributeType = service.getConceptAttributeTypeByUuid(UUID);
+        Assert.assertNotNull(conceptAttributeType);
+        Assert.assertFalse(conceptAttributeType.isRetired());
+
+        MockHttpServletRequest req = request(RequestMethod.DELETE, getURI() + "/" + UUID);
+        req.addParameter("!purge", "");
+        req.addParameter("reason", "let it retire for a time");
+        handle(req);
+
+        conceptAttributeType = service.getConceptAttributeTypeByUuid(UUID);
+        Assert.assertTrue(conceptAttributeType.isRetired());
```

Comment:
@Muta-Jonathan  you could do the same here.

Reply:
Thanks @Irenyak1...have updated it